modules/gov/utils_proposal.go

```diff
@@ -43,40 +44,67 @@ func (m *Module) UpdateProposal(height int64, blockTime time.Time, id uint64) er
 		return fmt.Errorf("error while updating proposal status: %s", err)
 	}
 
-	err = m.updateProposalTallyResult(proposal)
+	err = m.handlePassedProposal(proposal, height)
 	if err != nil {
-		return fmt.Errorf("error while updating proposal tally result: %s", err)
+		return fmt.Errorf("error while handling passed proposals: %s", err)
 	}
 
-	err = m.updateAccounts(proposal)
```

Comment:
May I know what is the purpose of removing this? :o

Reply:
This function was used to store the recipient address from `MsgCommunityPoolSpend` and `MsgExecLegacyContent` msgs, I moved it to `handleMsgSubmitProposal` handler, so we only need to process it once when the proposal is submitted instead of processing it on every block